config/packages/jms_serializer.yaml

```diff
@@ -0,0 +1,15 @@
+jms_serializer:
+    property_naming:
+        id: 'jms_serializer.identical_property_naming_strategy'
```

Comment:
If we want camelcase why don't we use this jms_serializer.camel_case_naming_strategy

Reply:
Good question, but this was the naming strategy we had before. I think this strategy will name the fields the same as in the entity, but not 100% sure.

Reply:
Also we didn't use true CamelCase.

Reply:
Also, also... Found this while trying to pull DOI citation in publinker. But other API's might be affected too. (Or not anymore now).